javatests/google/registry/flows/domain/DomainCreateFlowTest.java

```diff
@@ -437,6 +437,18 @@ public void testSuccess_generalAvailability_ignoresEncodedSignedMark() throws Ex
   }
 
   @Test
+  public void testSuccess_superuser_ignoresEncodedSignedMark() throws Exception {
```

Comment:
I believe you misread the change in the DomainCreateFlow. That one line change was moving the validation of the SMD into the 'isSuperuser' block. This also allows the superuser to not pass the SMD.